tests/functional/conftest.py

```diff
@@ -32,22 +37,30 @@ def pytest_configure(config):
 
         # User credentials
         user_username = "local-custom-model-templates-tests-{}@datarobot.com".format(suffix)
-        user_api_token = "lkjkljnm988989jkr5645tv_{}".format(suffix)
-        user_permissions = get_permissions("tests/fixtures/user_permissions.json", user_api_token)
+        user_password = "lkjkljnm988989jkr5645tv_{}".format(suffix)
+        user_api_key_name = "drum-functional-tests"
+        user_permissions = get_permissions(
+            "tests/fixtures/user_permissions.json", user_api_key_name
+        )
 
         # Add user
         DataRobotUserDatabase.add_user(
             db,
             env,
             user_username,
             invite_code="autogen",
+            password=user_password,
             app_user_manager=False,
             permissions=user_permissions,
-            api_token=user_api_token,
+            api_key_name=user_api_key_name,
             activated=True,
             unix_user="datarobot_imp",
         )
-        os.environ["DATAROBOT_API_TOKEN"] = user_api_token
+
+        user_api_keys = DataRobotUserDatabase.get_api_keys(env, user_username, user_password)
+        user_api_key = user_api_keys["data"][0]["key"]
+
+        os.environ["DATAROBOT_API_TOKEN"] = user_api_key
```

Comment:
eventually should replace the name with `DATAROBOT_API_KEY` for clarity

Reply:
added TODO, will merge this PR as is